src/sequence_batch_scheduler.cc

```diff
@@ -128,8 +145,8 @@ SequenceBatchScheduler::Create(
   const size_t model_batch_size = std::max(1, config.max_batch_size());
   sched->seq_slot_cnt_ = model_batch_size;
   if (config.sequence_batching().has_oldest()) {
-    sched->seq_slot_cnt_ =
-        config.sequence_batching().oldest().max_candidate_sequences();
+    sched->seq_slot_cnt_ = std::max(
+        1, config.sequence_batching().oldest().max_candidate_sequences());
```

Comment:
NOTE: This is just a Quality-Of-Life update because I noticed the default max_candidate_sequences would be zero if left unspecified with minimal config, and no error is reported to alert user:
```
sequence_batching {
  oldest { }
}
```
And the model would never run any requests.

Reply:
I think we need to set it to `model_batch_size` if it is zero.